src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
-<h1 data-test="header">Team Events</h1>
+<h1 data-test="header" class="header">Team Events</h1>
```

Comment:
Do we need this class? I don't see any CSS for this one.

Reply:
I don't know but the data-test="header" I use in test. Maybe later we'll use this class ;-)

Reply:
I change reference in css to class header from "h1"